aunt-leahs-app/src/components/select-box/select-box.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import { InputLabel, MenuItem, FormControl, Select } from '@material-ui/core';
+
+class SelectBox extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            items: this.props.items || [],
+            selectedItem: this.props.items && this.props.items[0],
+        }
+    }
+
+    render() {
+        return <div>
+            <FormControl fullWidth={true}>
+                <InputLabel id="volunteer-select-label">{this.props.name || "Select"}</InputLabel>
+                <Select
+                    labelId="volunteer-select-label"
+                    id="my-select"
+                    value={this.selectedItem}
+                    onChange={this.selectItem}
+                >
+                    <MenuItem value="">
```

Comment:
Do you have to give MenuItem a value?

Reply:
Yup that is the default value for "None" volunteer